contrib/gcppubsub/config/rbac_role_binding.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   creationTimestamp: null
-  name: gcppubsub-controller
+  name: gcppubsub-controller-rolebinding
```

Comment:
FWIW I find the `-rolebinding` suffix redundant.

Reply:
You'd need to update the vendor:

https://github.com/knative/eventing-sources/blob/master/vendor/sigs.k8s.io/controller-tools/pkg/generate/rbac/manifests.go#L52